README.md

```diff
@@ -35,31 +35,31 @@ func main() {
   // when there is an overflow
   //
   fmt.Println(safecast.ToInt8(float64(20000)))
-  // Output: 0 conversion issue: 20000 is greater than 127
+  // Output: 0 conversion issue: 20000.0 (float64) is greater than 127 (int8): maximum value for this type exceeded
```

Comment:
how do you keep this in sync with what the code does?

also not seeing that .0 when printing a float64 whose value is 20_000 with %v (which I think is what error uses below)

Reply:
> how do you keep this in sync with what the code does?

A copy pasta party from the example I'm running locally. But maybe I should simply copy the content on this README.md section, and launch it. To copy and paste the results

You are right about the float64, either there is a problem with the first 20000, or the fmt.Sprintf logic of %v is doing something strange. Anyway, I will change the example to use 20000.42, so there will be no problem.